kotlinpoet/src/test/java/com/squareup/kotlinpoet/ParameterizedTypeNameTest.kt

```diff
@@ -112,6 +112,15 @@ class ParameterizedTypeNameTest {
     assertThat(typeName.toString()).isEqualTo("java.util.Map<java.lang.String, java.lang.Integer>")
   }
 
+  @Test fun copyingTypeArguments() {
+    java.util.Map::class.java.plusParameter(java.lang.String::class.java)
```

Comment:
Does this line do anything?

Reply:
Damn, I left some debug stuff. But we can pretend I left it there to check if you would notice.

Reply:
I passed!